src/preferences/dialog/dlgprefdeck.cpp

```diff
@@ -106,6 +106,24 @@ DlgPrefDeck::DlgPrefDeck(QWidget * parent, MixxxMainWindow * mixxx,
     connect(buttonGroupTrackTime, SIGNAL(buttonClicked(QAbstractButton*)),
             this, SLOT(slotSetTrackTimeDisplay(QAbstractButton *)));
 
+    // display time format
+
+    m_pControlTrackTimeFormat = new ControlObject(
+            ConfigKey("[Controls]", "TimeFormat"));
+    connect(m_pControlTrackTimeDisplay, SIGNAL(valueChanged(double)),
+            this, SLOT(slotTimeFormatChanged(double)));
+
+    // Track Display model
+    comboBoxTimeFormat->clear();
+    comboBoxTimeFormat->addItem(tr("hh:mm:ss.zzz"),
+                                static_cast<int>(TrackTime::DisplayFormat::DEFAULT));
+    comboBoxTimeFormat->addItem(tr("k.sss:zz"),
```

Comment:
Or plain ks like use k.ssszz Or "k.sss zz" with space as group separator as defined in the SI system 

https://en.wikipedia.org/wiki/Decimal_separator#Digit_grouping

Reply:
Mmm I have just checked, currently I have the 00:00.00 format even in German. Now I can remember that we have discussed it before to use 00:00,00 for us, but this looks unusual, because all hardware devices have the dot, even though it would be correct. 

When reading your PR I was stumbeling over "k.sss:zzz", just because is somehow ambiguous in this case.

7.332:84 Colon 
7,332 84 Space
7,332 84 Punctuation Space
7,332 84 Thin Space
7,332 84 Hair Space

I think I prefer for now the localized thin Space version as recommended by SI-System.